functions/package-lock.json

```diff
@@ -3321,6 +3329,22 @@
         "@firebase/remote-config": "0.1.30",
         "@firebase/storage": "0.4.2",
         "@firebase/util": "0.3.4"
+      },
+      "dependencies": {
+        "@firebase/database": {
+          "version": "0.9.0",
+          "resolved": "https://registry.npmjs.org/@firebase/database/-/database-0.9.0.tgz",
+          "integrity": "sha512-j9NgRCsoahvAtQ8LPxMidGsiwsklbtUlDOQ6MhU6/Hk9DVYtEnVT3qSe42xOwBGDfMe191b1B2HsiEOsglWgag==",
+          "requires": {
+            "@firebase/auth-interop-types": "0.1.5",
+            "@firebase/component": "0.1.21",
+            "@firebase/database-types": "0.6.1",
+            "@firebase/logger": "0.2.6",
+            "@firebase/util": "0.3.4",
+            "faye-websocket": "0.11.3",
+            "tslib": "^1.11.1"
+          }
+        }
```

Comment:
It seems `firebase-admin` is still pinned to `@firebase/database` `0.8.1`, which is why we are now getting this additional version added to the lockfile:

```
..snip..
"firebase-admin": {
      "version": "9.4.2",
      "resolved": "https://registry.npmjs.org/firebase-admin/-/firebase-admin-9.4.2.tgz",
      "integrity": "sha512-mRnBJbW6BAz6DJkZ0GOUTkmnmCrwVzMreMc6O+RXWukFydOzi5Xr6TKSiPKxoOQw41r9IluP2AZ3Qzvlx2SR+g==",
      "requires": {
        "@firebase/database": "^0.8.1",
        "@firebase/database-types": "^0.6.1",
        ..snip..
```